src/test/java/steps/Meetings.java

```diff
@@ -139,9 +140,14 @@ public void anErrorMessageShouldBeDisplayed(String nameError){
 
     @And("^the Meeting should not be listed in the meetings of Room using the API$")
     public void theMeetingShouldNotBeListedInTheMeetingsOfRoomUsingTheAPI(){
-        assertTrue(true);
+        assertFalse(apiMeetingMethods.isMeetingPresent(meetingEntity.getSubject(),"title", mainTabletPage.getMainTitle()));
     }
 
+    @And("^I have a Meeting with the following information: \"([^\"]*)\", \"([^\"]*)\", \"([^\"]*)\", \"([^\"]*)\", \"([^\"]*)\", \"([^\"]*)\"$")
+    public void iHaveAMeetingWithTheFollowingInformation(String organizer, String subject, String from, String to, String attendees, String body){
+        meetingEntity.setAllFields(organizer, subject, from, to, attendees, body);
+        apiMeetingMethods.postMeeting(meetingEntity, nameRoomInMainTablet);
```

Comment:
nameRoomInMainTablet is the room that you use? if the answer is yes then you should to use a Entity for the room